drivers/adin2111/bm_adin2111.c

```diff
@@ -127,35 +127,84 @@ static BmErr adin2111_netdevice_enable(void) {
   return err;
 }
 
+static inline adin2111_Port_e adin2111_port_convert(uint8_t port) {
```

Comment:
I think we should rename this to `driver_port` and move the existing `driver_tx_port` function up here so that they're right next to each other. The goal is to make obvious at a glance that the underlying driver has multiple enums that represent ports, and that one may need to choose the right conversion function appropriate to the needed type. Also maybe add comments to further clarify what we're converting from and to. The public API of the network device trait should always have 0 representing all ports and 1-15 representing a specific port number.